### DISCUSSION TITLE
I'm unable to switch Google Accounts using Vieb

### DISCUSSION THREAD

**Discussion by uchoa** [Jelmerro/Vieb#543] [Q&A]
I'm having issues switching Google Accounts. It looks like something is blocking the feature. I've searched the Internet for a solution, most likely a configuration, but couldn't find anything. Any clues?

**Comment by Jelmerro** [comment]
See the FAQ: https://github.com/Jelmerro/Vieb/blob/master/FAQ.md#why-cant-i-sign-in-to-google basically google are trying to block it as much as possible for any browser that is not google's chrome. There are some suggestions in the FAQ item, but you decide how to solve it in the end :wink:
